example_configs/tomcat.yml

```diff
@@ -1,33 +1,680 @@
 ---   
+# Example jmx_exporter configuration for use with Tomcat 8.
+
 lowercaseOutputLabelNames: true
 lowercaseOutputName: true
-rules:
-- pattern: 'Catalina<type=GlobalRequestProcessor, name=\"(\w+-\w+)-(\d+)\"><>(\w+):'
-  name: tomcat_$3_total
+
+whitelistObjectNames: ["Tomcat:*", "java.lang:*"]
```

Comment:
Ok, I'll drop them all.

Some are important to me, but I understand it doesn't belong here.